src/bindings/python/src/openvino/opset16/__init__.py

```diff
@@ -4,6 +4,7 @@
 
 # New operations added in Opset16
 from openvino.opset16.ops import identity
+from openvino.opset16.ops import segment_max
```

Comment:
As we agreed, please update the remaining __init__ files, until we finalize opset16 and fully drop the `runtime` support: 
`src/bindings/python/src/openvino/runtime/opset16/__init__.py`
`src/bindings/python/src/openvino/runtime/opset16/ops/__init__.py`